src/services/logger/index.ts

```diff
@@ -11,26 +12,19 @@ const options = {
   reporters: {
     console: [
       {
-        module: 'good-squeeze',
-        name: 'Squeeze',
-        args: [{ log: '*', response: '*' }],
-      },
-      {
-        module: 'good-console',
+        module: GoodLogger,
```

Comment:
Can we remove `good-squeeze` from our modules?

Reply:
Yes